### DISCUSSION TITLE
omWeather and Breezy Weather show different daily forecast

### DISCUSSION THREAD

**Discussion by bege10** [breezy-weather/breezy-weather#438] [Q&A] [answered]
When in Breezy Weather the weather service is set to Open Meteo it shows a different forecast for the same places than omWeather which uses the same service. How come?
https://github.com/woheller69/omweather/issues/46

**Comment by papjul** [comment]
Please give more details: screenshots, examples, what kind of data is impacted, etc

**Comment by papjul** [comment]
I see no difference:
![IMG_20230902_074630](https://github.com/breezy-weather/breezy-weather/assets/1449159/95244674-0b7b-4c43-a47c-33b744828e6d)
![IMG_20230902_074559](https://github.com/breezy-weather/breezy-weather/assets/1449159/98756421-8d6a-4448-92bb-a8298ab202f4)
![IMG_20230902_074615](https://github.com/breezy-weather/breezy-weather/assets/1449159/b23fd7ec-e779-4cc2-ab6c-4e7047ea87df)

**Comment by bege10** [comment]
This is what I see right now. The differences are only slight now, in the last weeks with rapidly changing weather they were bigger. This shows differences in temperature. When it was raining in the last days there were differences also.

![Screenshot_20230903-154008_omWeather](https://github.com/breezy-weather/breezy-weather/assets/1184237/a7dd0118-6ef3-4f00-bb62-92c2857d32f5)
![Screenshot_20230903-154036_Breezy Weather](https://github.com/breezy-weather/breezy-weather/assets/1184237/fa40dd14-b87e-4a00-b3b5-4807af987693)
![Screenshot_20230903-154053_Breezy Weather](https://github.com/breezy-weather/breezy-weather/assets/1184237/3d103637-5b34-4100-af83-1ae58424e8fd)

**From the Open Meteo website:**  

![Screenshot_334](https://github.com/breezy-weather/breezy-weather/assets/1184237/08f34ca0-e64f-43d6-ba0b-8bb009b71e4b)

**Comment by papjul** [comment, marked as answer]
I don't know what exactly is the issue, but you're comparing things that are not the same.
We use half days in Breezy Weather. You can read more about it here: https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md#daily-forecast

**Comment by bege10** [comment]
So you don't take the daily temperature from Open Meteo but calculate it by the hourly data from 6:00 to 6:00? That may create differences in the minimum temperature but not in the maximum temperature, as far as I understand.

**Reply by papjul** [reply]
I see no discrepancy between hourly forecast minimum and maximum for that day and min/max temperature for that day,
Do you have an example?

**Reply by bege10** [reply]
..........................Breezy Weather.....................omWeather....................Open Meteo

3.9. (today).............20/10...................................20/11.....................................20/12
4.9..............................24/13...................................24/10.....................................24/10
5.9..............................27/14...................................28/13.....................................27/13
6.9..............................28/15...................................29/14.....................................28/14

So indeed the maximum temperature is the same as in Open Meteo, the minimum temperature is different.

(Unfortunately the spaces between the columns don't remain after saving.)

**Reply by papjul** [reply]
I can't see your table. Did you pick your minimum temperature from overnight, not from the night before?

Please add screenshots from the app instead, they are much more easier to understand.

**Reply by bege10** [reply]
I took the data in the table from the screenshots above. Can't you see them either?

**Reply by papjul** [reply]
No, I don't see any hourly forecast screenshot :/

**Reply by bege10** [reply]
Oh, I just thought that that kind of calculation of day and night could be the reason for the differences. I did not take the hourly data, I just compared the data of the days in the two apps and Open Meteo. AccuWeather does the same calculation as Breezy Weather. Most apps I compared just take the 24 hours of the day, but that is the decision of the app developers. Because the maximum temperature of Breezy Weather is the same as in Open Meteo I think it is OK now and I understand why the two apps show different data with the same data underlying.

**Reply by bege10** [reply]
P.S. I edited the table, now it is easier to read.

**Reply by papjul** [reply]
Yes, this project is forked from Geometric Weather which had AccuWeather and a China sources that worked this way. And honestly, I feel like they are more intuitive, although it can be confusing at first. When looking at a day forecast, I want to know how the night will be, not how the previous night went.

**Reply by bege10** [reply]
That's a good argument. This is just a case of what one is used to. Anyway, it is a good looking and feature rich app. Thank you.
